From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 11 Dec 2017 22:42:11 +0100
Subject: Add English-only search engine

Add a Google search engine that forces languages to English,
disable from all its searches RLZ and field experiments querystring parameters.
---
 .../search_engines/prepopulated_engines.json  |  21 ++
 .../search_engines/search_engine_type.h       |   1 +
 .../template_url_prepopulate_data.cc          | 216 +++++++++---------
 3 files changed, 130 insertions(+), 108 deletions(-)

diff --git a/components/search_engines/prepopulated_engines.json b/components/search_engines/prepopulated_engines.json
--- a/components/search_engines/prepopulated_engines.json
+++ b/components/search_engines/prepopulated_engines.json
@@ -134,6 +134,27 @@
       "id": 1
     },
 
+    "googleen": {
+      "name": "Google in English",
+      "keyword": "googleen",
+      "favicon_url": "https://www.google.com/favicon.ico",
+      "search_url": "{google:baseURL}search?q={searchTerms}&ie={inputEncoding}&hl=en",
+      "suggest_url": "{google:baseSuggestURL}search?client={google:suggestClient}&q={searchTerms}&hl=en",
+      "image_url": "{google:baseURL}searchbyimage/upload?hl=en",
+      "new_tab_url": "{google:baseURL}_/chrome/newtab?hl=en&ie={inputEncoding}",
+      "contextual_search_url": "{google:baseURL}_/contextualsearch?{google:contextualSearchVersion}{google:contextualSearchContextData}&hl=en",
+      "image_url_post_params": "encoded_image={google:imageThumbnail},image_url={google:imageURL},sbisrc={google:imageSearchSource},original_width={google:imageOriginalWidth},original_height={google:imageOriginalHeight}",
+      "alternate_urls": [
+        "{google:baseURL}?hl=en#q={searchTerms}",
+        "{google:baseURL}search?hl=en#q={searchTerms}",
+        "{google:baseURL}webhp?hl=en#q={searchTerms}",
+        "{google:baseURL}s?hl=en#q={searchTerms}",
+        "{google:baseURL}s?hl=en&q={searchTerms}"
+      ],
+      "type": "SEARCH_ENGINE_GOOGLE_EN",
+      "id": 13
+    },
+
     "mail_ru": {
       "name": "@MAIL.RU",
       "keyword": "mail.ru",
diff --git a/components/search_engines/search_engine_type.h b/components/search_engines/search_engine_type.h
--- a/components/search_engines/search_engine_type.h
+++ b/components/search_engines/search_engine_type.h
@@ -75,6 +75,7 @@ enum SearchEngineType {
   SEARCH_ENGINE_PRIVACYWALL = 58,
   SEARCH_ENGINE_ECOSIA = 59,
   SEARCH_ENGINE_PETALSEARCH = 60,
+  SEARCH_ENGINE_GOOGLE_EN = 61,
 
   SEARCH_ENGINE_MAX  // Bounding value needed for UMA histogram macro.
 };
diff --git a/components/search_engines/template_url_prepopulate_data.cc b/components/search_engines/template_url_prepopulate_data.cc
--- a/components/search_engines/template_url_prepopulate_data.cc
+++ b/components/search_engines/template_url_prepopulate_data.cc
@@ -28,7 +28,7 @@ namespace {
 
 // Default (for countries with no better engine set)
 const PrepopulatedEngine* const engines_default[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
 };
@@ -38,7 +38,7 @@ const PrepopulatedEngine* const engines_default[] = {
 // clang-format off
 // United Arab Emirates
 const PrepopulatedEngine* const engines_AE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -47,7 +47,7 @@ const PrepopulatedEngine* const engines_AE[] = {
 
 // Albania
 const PrepopulatedEngine* const engines_AL[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -56,7 +56,7 @@ const PrepopulatedEngine* const engines_AL[] = {
 
 // Argentina
 const PrepopulatedEngine* const engines_AR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_ar,
     &duckduckgo,
@@ -65,7 +65,7 @@ const PrepopulatedEngine* const engines_AR[] = {
 
 // Austria
 const PrepopulatedEngine* const engines_AT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo_at,
@@ -74,7 +74,7 @@ const PrepopulatedEngine* const engines_AT[] = {
 
 // Australia
 const PrepopulatedEngine* const engines_AU[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_au,
     &duckduckgo,
@@ -83,7 +83,7 @@ const PrepopulatedEngine* const engines_AU[] = {
 
 // Bosnia and Herzegovina
 const PrepopulatedEngine* const engines_BA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -92,7 +92,7 @@ const PrepopulatedEngine* const engines_BA[] = {
 
 // Belgium
 const PrepopulatedEngine* const engines_BE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &ecosia,
@@ -101,7 +101,7 @@ const PrepopulatedEngine* const engines_BE[] = {
 
 // Bulgaria
 const PrepopulatedEngine* const engines_BG[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -110,7 +110,7 @@ const PrepopulatedEngine* const engines_BG[] = {
 
 // Bahrain
 const PrepopulatedEngine* const engines_BH[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -119,7 +119,7 @@ const PrepopulatedEngine* const engines_BH[] = {
 
 // Burundi
 const PrepopulatedEngine* const engines_BI[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -128,7 +128,7 @@ const PrepopulatedEngine* const engines_BI[] = {
 
 // Brunei
 const PrepopulatedEngine* const engines_BN[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -137,7 +137,7 @@ const PrepopulatedEngine* const engines_BN[] = {
 
 // Bolivia
 const PrepopulatedEngine* const engines_BO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -146,7 +146,7 @@ const PrepopulatedEngine* const engines_BO[] = {
 
 // Brazil
 const PrepopulatedEngine* const engines_BR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_br,
     &duckduckgo,
@@ -155,7 +155,7 @@ const PrepopulatedEngine* const engines_BR[] = {
 
 // Belarus
 const PrepopulatedEngine* const engines_BY[] = {
-    &google,
+    &googleen, &google,
     &yandex_by,
     &mail_ru,
     &bing,
@@ -164,7 +164,7 @@ const PrepopulatedEngine* const engines_BY[] = {
 
 // Belize
 const PrepopulatedEngine* const engines_BZ[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -173,7 +173,7 @@ const PrepopulatedEngine* const engines_BZ[] = {
 
 // Canada
 const PrepopulatedEngine* const engines_CA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_ca,
     &duckduckgo,
@@ -182,7 +182,7 @@ const PrepopulatedEngine* const engines_CA[] = {
 
 // Switzerland
 const PrepopulatedEngine* const engines_CH[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &ecosia,
@@ -191,7 +191,7 @@ const PrepopulatedEngine* const engines_CH[] = {
 
 // Chile
 const PrepopulatedEngine* const engines_CL[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_cl,
     &duckduckgo,
@@ -203,13 +203,13 @@ const PrepopulatedEngine* const engines_CN[] = {
     &sogou,
     &baidu,
     &bing,
-    &google,
+    &googleen, &google,
     &so_360,
 };
 
 // Colombia
 const PrepopulatedEngine* const engines_CO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_co,
     &ecosia,
@@ -218,7 +218,7 @@ const PrepopulatedEngine* const engines_CO[] = {
 
 // Costa Rica
 const PrepopulatedEngine* const engines_CR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -227,7 +227,7 @@ const PrepopulatedEngine* const engines_CR[] = {
 
 // Czech Republic
 const PrepopulatedEngine* const engines_CZ[] = {
-    &google,
+    &googleen, &google,
     &seznam_cz,
     &bing,
     &yahoo,
@@ -236,7 +236,7 @@ const PrepopulatedEngine* const engines_CZ[] = {
 
 // Germany
 const PrepopulatedEngine* const engines_DE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &ecosia,
     &duckduckgo,
@@ -245,7 +245,7 @@ const PrepopulatedEngine* const engines_DE[] = {
 
 // Denmark
 const PrepopulatedEngine* const engines_DK[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_dk,
     &duckduckgo,
@@ -254,7 +254,7 @@ const PrepopulatedEngine* const engines_DK[] = {
 
 // Dominican Republic
 const PrepopulatedEngine* const engines_DO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -263,7 +263,7 @@ const PrepopulatedEngine* const engines_DO[] = {
 
 // Algeria
 const PrepopulatedEngine* const engines_DZ[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_fr,
     &yandex_com,
@@ -272,7 +272,7 @@ const PrepopulatedEngine* const engines_DZ[] = {
 
 // Ecuador
 const PrepopulatedEngine* const engines_EC[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &ecosia,
@@ -281,7 +281,7 @@ const PrepopulatedEngine* const engines_EC[] = {
 
 // Estonia
 const PrepopulatedEngine* const engines_EE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yandex_ru,
     &duckduckgo,
@@ -290,7 +290,7 @@ const PrepopulatedEngine* const engines_EE[] = {
 
 // Egypt
 const PrepopulatedEngine* const engines_EG[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_com,
@@ -299,7 +299,7 @@ const PrepopulatedEngine* const engines_EG[] = {
 
 // Spain
 const PrepopulatedEngine* const engines_ES[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -308,7 +308,7 @@ const PrepopulatedEngine* const engines_ES[] = {
 
 // Finland
 const PrepopulatedEngine* const engines_FI[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_fi,
     &duckduckgo,
@@ -317,7 +317,7 @@ const PrepopulatedEngine* const engines_FI[] = {
 
 // Faroe Islands
 const PrepopulatedEngine* const engines_FO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_dk,
     &duckduckgo,
@@ -326,7 +326,7 @@ const PrepopulatedEngine* const engines_FO[] = {
 
 // France
 const PrepopulatedEngine* const engines_FR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_fr,
     &qwant,
@@ -335,7 +335,7 @@ const PrepopulatedEngine* const engines_FR[] = {
 
 // United Kingdom
 const PrepopulatedEngine* const engines_GB[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_uk,
     &duckduckgo,
@@ -344,7 +344,7 @@ const PrepopulatedEngine* const engines_GB[] = {
 
 // Greece
 const PrepopulatedEngine* const engines_GR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -353,7 +353,7 @@ const PrepopulatedEngine* const engines_GR[] = {
 
 // Guatemala
 const PrepopulatedEngine* const engines_GT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -362,7 +362,7 @@ const PrepopulatedEngine* const engines_GT[] = {
 
 // Hong Kong
 const PrepopulatedEngine* const engines_HK[] = {
-    &google,
+    &googleen, &google,
     &yahoo_hk,
     &bing,
     &baidu,
@@ -371,7 +371,7 @@ const PrepopulatedEngine* const engines_HK[] = {
 
 // Honduras
 const PrepopulatedEngine* const engines_HN[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -380,7 +380,7 @@ const PrepopulatedEngine* const engines_HN[] = {
 
 // Croatia
 const PrepopulatedEngine* const engines_HR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -389,7 +389,7 @@ const PrepopulatedEngine* const engines_HR[] = {
 
 // Hungary
 const PrepopulatedEngine* const engines_HU[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -398,7 +398,7 @@ const PrepopulatedEngine* const engines_HU[] = {
 
 // Indonesia
 const PrepopulatedEngine* const engines_ID[] = {
-    &google,
+    &googleen, &google,
     &yahoo_id,
     &bing,
     &yandex_com,
@@ -407,7 +407,7 @@ const PrepopulatedEngine* const engines_ID[] = {
 
 // Ireland
 const PrepopulatedEngine* const engines_IE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_uk,
     &duckduckgo,
@@ -416,7 +416,7 @@ const PrepopulatedEngine* const engines_IE[] = {
 
 // Israel
 const PrepopulatedEngine* const engines_IL[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yandex_ru,
     &yahoo,
@@ -425,7 +425,7 @@ const PrepopulatedEngine* const engines_IL[] = {
 
 // India
 const PrepopulatedEngine* const engines_IN[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_in,
     &duckduckgo,
@@ -434,7 +434,7 @@ const PrepopulatedEngine* const engines_IN[] = {
 
 // Iraq
 const PrepopulatedEngine* const engines_IQ[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_tr,
@@ -443,7 +443,7 @@ const PrepopulatedEngine* const engines_IQ[] = {
 
 // Iran
 const PrepopulatedEngine* const engines_IR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &ask,
@@ -452,7 +452,7 @@ const PrepopulatedEngine* const engines_IR[] = {
 
 // Iceland
 const PrepopulatedEngine* const engines_IS[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo,
@@ -461,7 +461,7 @@ const PrepopulatedEngine* const engines_IS[] = {
 
 // Italy
 const PrepopulatedEngine* const engines_IT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -470,7 +470,7 @@ const PrepopulatedEngine* const engines_IT[] = {
 
 // Jamaica
 const PrepopulatedEngine* const engines_JM[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -479,7 +479,7 @@ const PrepopulatedEngine* const engines_JM[] = {
 
 // Jordan
 const PrepopulatedEngine* const engines_JO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -488,7 +488,7 @@ const PrepopulatedEngine* const engines_JO[] = {
 
 // Japan
 const PrepopulatedEngine* const engines_JP[] = {
-    &google,
+    &googleen, &google,
     &yahoo_jp,
     &bing,
     &duckduckgo,
@@ -497,7 +497,7 @@ const PrepopulatedEngine* const engines_JP[] = {
 
 // Kenya
 const PrepopulatedEngine* const engines_KE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -506,7 +506,7 @@ const PrepopulatedEngine* const engines_KE[] = {
 
 // South Korea
 const PrepopulatedEngine* const engines_KR[] = {
-    &google,
+    &googleen, &google,
     &naver,
     &daum,
     &bing,
@@ -515,7 +515,7 @@ const PrepopulatedEngine* const engines_KR[] = {
 
 // Kuwait
 const PrepopulatedEngine* const engines_KW[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -524,7 +524,7 @@ const PrepopulatedEngine* const engines_KW[] = {
 
 // Kazakhstan
 const PrepopulatedEngine* const engines_KZ[] = {
-    &google,
+    &googleen, &google,
     &yandex_kz,
     &mail_ru,
     &bing,
@@ -533,7 +533,7 @@ const PrepopulatedEngine* const engines_KZ[] = {
 
 // Lebanon
 const PrepopulatedEngine* const engines_LB[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -542,7 +542,7 @@ const PrepopulatedEngine* const engines_LB[] = {
 
 // Liechtenstein
 const PrepopulatedEngine* const engines_LI[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo,
@@ -551,7 +551,7 @@ const PrepopulatedEngine* const engines_LI[] = {
 
 // Lithuania
 const PrepopulatedEngine* const engines_LT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -560,7 +560,7 @@ const PrepopulatedEngine* const engines_LT[] = {
 
 // Luxembourg
 const PrepopulatedEngine* const engines_LU[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo,
@@ -569,7 +569,7 @@ const PrepopulatedEngine* const engines_LU[] = {
 
 // Latvia
 const PrepopulatedEngine* const engines_LV[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yandex_ru,
     &yahoo,
@@ -578,7 +578,7 @@ const PrepopulatedEngine* const engines_LV[] = {
 
 // Libya
 const PrepopulatedEngine* const engines_LY[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_com,
@@ -587,7 +587,7 @@ const PrepopulatedEngine* const engines_LY[] = {
 
 // Morocco
 const PrepopulatedEngine* const engines_MA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_fr,
     &yandex_com,
@@ -596,7 +596,7 @@ const PrepopulatedEngine* const engines_MA[] = {
 
 // Monaco
 const PrepopulatedEngine* const engines_MC[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo_fr,
@@ -605,7 +605,7 @@ const PrepopulatedEngine* const engines_MC[] = {
 
 // Moldova
 const PrepopulatedEngine* const engines_MD[] = {
-    &google,
+    &googleen, &google,
     &yandex_ru,
     &bing,
     &mail_ru,
@@ -614,7 +614,7 @@ const PrepopulatedEngine* const engines_MD[] = {
 
 // Montenegro
 const PrepopulatedEngine* const engines_ME[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_ru,
@@ -623,7 +623,7 @@ const PrepopulatedEngine* const engines_ME[] = {
 
 // Macedonia
 const PrepopulatedEngine* const engines_MK[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -632,7 +632,7 @@ const PrepopulatedEngine* const engines_MK[] = {
 
 // Mexico
 const PrepopulatedEngine* const engines_MX[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_mx,
     &duckduckgo,
@@ -641,7 +641,7 @@ const PrepopulatedEngine* const engines_MX[] = {
 
 // Malaysia
 const PrepopulatedEngine* const engines_MY[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_my,
     &duckduckgo,
@@ -650,7 +650,7 @@ const PrepopulatedEngine* const engines_MY[] = {
 
 // Nicaragua
 const PrepopulatedEngine* const engines_NI[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -659,7 +659,7 @@ const PrepopulatedEngine* const engines_NI[] = {
 
 // Netherlands
 const PrepopulatedEngine* const engines_NL[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_nl,
     &duckduckgo,
@@ -668,7 +668,7 @@ const PrepopulatedEngine* const engines_NL[] = {
 
 // Norway
 const PrepopulatedEngine* const engines_NO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -677,7 +677,7 @@ const PrepopulatedEngine* const engines_NO[] = {
 
 // New Zealand
 const PrepopulatedEngine* const engines_NZ[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_nz,
     &duckduckgo,
@@ -686,7 +686,7 @@ const PrepopulatedEngine* const engines_NZ[] = {
 
 // Oman
 const PrepopulatedEngine* const engines_OM[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -695,7 +695,7 @@ const PrepopulatedEngine* const engines_OM[] = {
 
 // Panama
 const PrepopulatedEngine* const engines_PA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -704,7 +704,7 @@ const PrepopulatedEngine* const engines_PA[] = {
 
 // Peru
 const PrepopulatedEngine* const engines_PE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_pe,
     &ecosia,
@@ -713,7 +713,7 @@ const PrepopulatedEngine* const engines_PE[] = {
 
 // Philippines
 const PrepopulatedEngine* const engines_PH[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &ecosia,
@@ -722,7 +722,7 @@ const PrepopulatedEngine* const engines_PH[] = {
 
 // Pakistan
 const PrepopulatedEngine* const engines_PK[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -731,7 +731,7 @@ const PrepopulatedEngine* const engines_PK[] = {
 
 // Poland
 const PrepopulatedEngine* const engines_PL[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -740,7 +740,7 @@ const PrepopulatedEngine* const engines_PL[] = {
 
 // Puerto Rico
 const PrepopulatedEngine* const engines_PR[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -749,7 +749,7 @@ const PrepopulatedEngine* const engines_PR[] = {
 
 // Portugal
 const PrepopulatedEngine* const engines_PT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -758,7 +758,7 @@ const PrepopulatedEngine* const engines_PT[] = {
 
 // Paraguay
 const PrepopulatedEngine* const engines_PY[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -767,7 +767,7 @@ const PrepopulatedEngine* const engines_PY[] = {
 
 // Qatar
 const PrepopulatedEngine* const engines_QA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -776,7 +776,7 @@ const PrepopulatedEngine* const engines_QA[] = {
 
 // Romania
 const PrepopulatedEngine* const engines_RO[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -785,7 +785,7 @@ const PrepopulatedEngine* const engines_RO[] = {
 
 // Serbia
 const PrepopulatedEngine* const engines_RS[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -794,7 +794,7 @@ const PrepopulatedEngine* const engines_RS[] = {
 
 // Russia
 const PrepopulatedEngine* const engines_RU[] = {
-    &google,
+    &googleen, &google,
     &yandex_ru,
     &mail_ru,
     &bing,
@@ -803,7 +803,7 @@ const PrepopulatedEngine* const engines_RU[] = {
 
 // Rwanda
 const PrepopulatedEngine* const engines_RW[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -812,7 +812,7 @@ const PrepopulatedEngine* const engines_RW[] = {
 
 // Saudi Arabia
 const PrepopulatedEngine* const engines_SA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_com,
@@ -821,7 +821,7 @@ const PrepopulatedEngine* const engines_SA[] = {
 
 // Sweden
 const PrepopulatedEngine* const engines_SE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_se,
     &duckduckgo,
@@ -830,7 +830,7 @@ const PrepopulatedEngine* const engines_SE[] = {
 
 // Singapore
 const PrepopulatedEngine* const engines_SG[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_sg,
     &duckduckgo,
@@ -839,7 +839,7 @@ const PrepopulatedEngine* const engines_SG[] = {
 
 // Slovenia
 const PrepopulatedEngine* const engines_SI[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo,
@@ -848,7 +848,7 @@ const PrepopulatedEngine* const engines_SI[] = {
 
 // Slovakia
 const PrepopulatedEngine* const engines_SK[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &duckduckgo,
     &yahoo,
@@ -857,7 +857,7 @@ const PrepopulatedEngine* const engines_SK[] = {
 
 // El Salvador
 const PrepopulatedEngine* const engines_SV[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -866,7 +866,7 @@ const PrepopulatedEngine* const engines_SV[] = {
 
 // Syria
 const PrepopulatedEngine* const engines_SY[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_com,
@@ -875,7 +875,7 @@ const PrepopulatedEngine* const engines_SY[] = {
 
 // Thailand
 const PrepopulatedEngine* const engines_TH[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_th,
     &yandex_ru,
@@ -884,7 +884,7 @@ const PrepopulatedEngine* const engines_TH[] = {
 
 // Tunisia
 const PrepopulatedEngine* const engines_TN[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_fr,
     &duckduckgo,
@@ -893,7 +893,7 @@ const PrepopulatedEngine* const engines_TN[] = {
 
 // Turkey
 const PrepopulatedEngine* const engines_TR[] = {
-    &google,
+    &googleen, &google,
     &yandex_tr,
     &yahoo_tr,
     &bing,
@@ -902,7 +902,7 @@ const PrepopulatedEngine* const engines_TR[] = {
 
 // Trinidad and Tobago
 const PrepopulatedEngine* const engines_TT[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -911,7 +911,7 @@ const PrepopulatedEngine* const engines_TT[] = {
 
 // Taiwan
 const PrepopulatedEngine* const engines_TW[] = {
-    &google,
+    &googleen, &google,
     &yahoo_tw,
     &bing,
     &baidu,
@@ -920,7 +920,7 @@ const PrepopulatedEngine* const engines_TW[] = {
 
 // Tanzania
 const PrepopulatedEngine* const engines_TZ[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -929,7 +929,7 @@ const PrepopulatedEngine* const engines_TZ[] = {
 
 // Ukraine
 const PrepopulatedEngine* const engines_UA[] = {
-    &google,
+    &googleen, &google,
     &yandex_ua,
     &bing,
     &duckduckgo,
@@ -938,7 +938,7 @@ const PrepopulatedEngine* const engines_UA[] = {
 
 // United States
 const PrepopulatedEngine* const engines_US[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -947,7 +947,7 @@ const PrepopulatedEngine* const engines_US[] = {
 
 // Uruguay
 const PrepopulatedEngine* const engines_UY[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -956,7 +956,7 @@ const PrepopulatedEngine* const engines_UY[] = {
 
 // Venezuela
 const PrepopulatedEngine* const engines_VE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo_es,
     &duckduckgo,
@@ -965,7 +965,7 @@ const PrepopulatedEngine* const engines_VE[] = {
 
 // Vietnam
 const PrepopulatedEngine* const engines_VN[] = {
-    &google,
+    &googleen, &google,
     &coccoc,
     &yahoo,
     &bing,
@@ -974,7 +974,7 @@ const PrepopulatedEngine* const engines_VN[] = {
 
 // Yemen
 const PrepopulatedEngine* const engines_YE[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &yandex_com,
@@ -983,7 +983,7 @@ const PrepopulatedEngine* const engines_YE[] = {
 
 // South Africa
 const PrepopulatedEngine* const engines_ZA[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
@@ -992,7 +992,7 @@ const PrepopulatedEngine* const engines_ZA[] = {
 
 // Zimbabwe
 const PrepopulatedEngine* const engines_ZW[] = {
-    &google,
+    &googleen, &google,
     &bing,
     &yahoo,
     &duckduckgo,
--
2.25.1
